src/commands/sbom/payload.ts

```diff
@@ -138,15 +126,15 @@ export const generatePayload = (
   return {
     id: crypto.randomUUID(),
     commit: {
-      author_name: tags[GIT_COMMIT_AUTHOR_NAME],
-      author_email: tags[GIT_COMMIT_AUTHOR_EMAIL],
-      committer_name: tags[GIT_COMMIT_COMMITTER_NAME],
-      committer_email: tags[GIT_COMMIT_COMMITTER_EMAIL],
-      sha: tags[GIT_SHA],
-      branch: tags[GIT_BRANCH],
+      author_name: tags[GIT_COMMIT_AUTHOR_NAME] ?? '',
```

Comment:
Nit: could we add a comment to indicate that these tags have a value and are checked earlier?